packages/react-router-native/NativeRouter.js

```diff
@@ -20,8 +20,6 @@ NativeRouter.defaultProps = {
   }
 };
 
-const __DEV__ = true; // TODO
```

Comment:
We don't actually have a build for `react-router-native` yet, so we still need this.

Reply:
Whoops, my bad. I thought I saw something but I guess I'm wrong 😊 Will fix.